admin/class-fastcgi-purger.php

```diff
@@ -184,18 +184,42 @@ public function purge_all() {
 	}
 
 	/**
-	 * Constructs the base url to call when purging using the `get_request` method.
+	 * Constructs the base url to call when purging using the "get_request" method.
+	 *
+	 * @since 2.2.0
 	 *
 	 * @return string
 	 */
 	private function purge_base_url() {
+
 		$parse = wp_parse_url( home_url() );
+
+		/**
+		 * Filter to change purge suffix for FastCGI cache.
+		 *
+		 * @param string $suffix Purge suffix. Default is purge.
+		 *
+		 * @since 2.2.0
+		 */
 		$path  = trim( apply_filters( 'rt_nginx_helper_fastcgi_purge_suffix', 'purge' ), '/' );
 
+		// Prevent users from inserting a trailing '/' that could break the url purging.
+		$path  = trim( $path, '/' );
```

Comment:
```suggestion
		$path  = untrailingslashit( $path, '/' );
```
Could we use `untrailingslashit`?

Reply:
`untrailingslashit()` will remove only from right side of the string. But I can use it in `return rtrim( $purge_url_base, '/' );`